crates/jiter-python/tests/emscripten_runner.js

```diff
@@ -0,0 +1,122 @@
+const {opendir} = require('node:fs/promises');
+const {loadPyodide} = require('pyodide');
+const path = require('path');
+
+async function find_wheel(dist_dir) {
+  const dir = await opendir(dist_dir);
+  for await (const dirent of dir) {
+    if (dirent.name.endsWith('.whl')) {
+      return path.join(dist_dir, dirent.name);
+    }
+  }
+}
+
+function make_tty_ops(stream) {
```

Comment:
all this crap with tty's will put people off. I suggest we keep running pyodide, but do the simple thing in terms of wiring up stdout and stderr, see https://github.com/pydantic/logfire/blob/main/pyodide_test/test.mjs